llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -347,6 +347,16 @@ bool isModuleUsingAsan(const Module &M) {
   return MDVal->getString() == "asan";
 }
 
+// Gets num_dim_reqd_work_group_size information for function Func, conviniently
+// 0 if metadata is not presetn.
```

Comment:
typo: `present`.

I also wonder if the default should be 3. That would preserve existing behaviour, I think, and mean that it is safe to remove the metadata, because the "missing" dimensions of the reqd are set to 1 anyway?

Depends on who uses this I suppose, I might be thinking too broadly.